.github/workflows/run-crasher.yml

```diff
@@ -70,10 +70,30 @@ jobs:
         run: |
           crashing_time="${{ steps.default_inputs.outputs.crashing_time }}"
           ./crash-things.sh "/home/runner/work/crasher/crasher/qdrant/" "target/debug/qdrant" "0.2" "$crashing_time"
-      - name: Upload logs in case of failure
+      - name: Upload logs on failure
         uses: actions/upload-artifact@v4
         if: failure() || cancelled()
         with:
           name: crasher-logs
           retention-days: 90 # default max value
-          path: crasher_output.log
\ No newline at end of file
+          path: crasher_output.log
+      - name: Send Notification
+        if: always()
```

Comment:
`always()` means that it also sends a notification in case of success?

Reply:
Yes, this is for debug purposes before merging.

Reply:
set back to `if: failure() || cancelled()`